### ISSUE TITLE
`displayName` is not set for the component of "@yamada-ui/number-input"

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#2888]
script: component-display-name
package: @yamada-ui/number-input

### Description
The component "@yamada-ui/number-input" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

**Comment by Kei-Ta** [comment]
@hirotomoyamada 
Can i try this issue?

**Comment by hirotomoyamada** [comment]
@Kei-Ta 

Hi, thanks for the comment!

Yes, pls!

**Comment by Kei-Ta** [comment]
@hirotomoyamada 
I think tasks for this issue have been closed by #3022 .
If there is no task, please close this issue.

**Comment by hirotomoyamada** [comment]
@Kei-Ta 

Oh, sorry...

Close this Issue.
